Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    amazon-pricing (0.1.78)
+    amazon-pricing (0.1.79)
```

Comment:
@stanmluke, since this repo was deployed to `0.1.79` before you merged, you probably need to bump to `0.1.80` instead.
Also, please squash commits like these next time :-)